htdocs/categories/class/categorie.class.php

```diff
@@ -753,6 +753,7 @@ function getObjectsInCateg($type)
 		$sql .= " WHERE o.entity IN (" . getEntity( $obj->element, 1).")";
 		$sql.= " AND c.fk_categorie = ".$this->id;
 		$sql .= " AND c.fk_" . $this->MAP_CAT_FK[$type] . " = o.rowid";
+		if ($field=='societe') $sql.= " ORDER by o.nom ASC";
```

Comment:
Variable $field is not defined. Also it would be better to sort after reception of result by the caller of function.
Having a sort criteria should be better into function that use the result.